jpa/eclipselink.jpa.testapps/jpa.test.persistence32/pom.xml

```diff
@@ -80,6 +80,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Static metamodel is used in tests -->
+            <plugin>
+                <groupId>org.bsc.maven</groupId>
+                <artifactId>maven-processor-plugin</artifactId>
+                <version>4.5</version>
```

Comment:
why? see ie [inheritance tests](https://github.com/eclipse-ee4j/eclipselink/blob/master/jpa/eclipselink.jpa.testapps/jpa.test.inheritance/pom.xml#L37-L49) how to use what we already have

Reply:
OK, fixed.